vendor/bat-native-ledger/src/bat_helper.cc

```diff
@@ -2618,6 +2619,24 @@ uint64_t getMediaDuration(const std::map<std::string, std::string>& data,
   return duration;
 }
 
+bool HasSameDomainAndPath(
+    const std::string& url_to_validate,
+    const std::string& domain_to_match,
+    const std::string& path_to_match) {
+  GURL gurl(url_to_validate);
+  GURL gurl_to_match(domain_to_match);
+  if (gurl.is_valid() && gurl_to_match.is_valid() &&
```

Comment:
`gurl_to_match.is_valid()` doesn't seem necessary; maybe just check that `domain_to_match` is non-empty

Reply:
Fixed